pkg/i18n/french.go

```diff
@@ -8,32 +8,32 @@ func frenchSet() TranslationSet {
 		StartingStatus:             "démarrage",
 		StoppingStatus:             "arrêt",
 		PausingStatus:              "mise en pause",
-		RunningCustomCommandStatus: "execution de la commande personalisée",
-		RunningBulkCommandStatus:   "execution de la commande groupée",
+		RunningCustomCommandStatus: "exécution de la commande personalisée",
+		RunningBulkCommandStatus:   "exécution de la commande groupée",
 
 		NoViewMachingNewLineFocusedSwitchStatement: "No view matching newLineFocused switch statement",
 
-		ErrorOccurred:              "Une erreur s'est produite! Veuillez créer un rapport d'erreur sur https://github.com/jesseduffield/lazydocker/issues",
+		ErrorOccurred:              "Une erreur s'est produite ! Veuillez créer un rapport d'erreur sur https://github.com/jesseduffield/lazydocker/issues",
 		ConnectionFailed:           "Erreur lors de la connexion au client docker. Essayez de redémarrer votre client docker",
-		UnattachableContainerError: "Le conteneur ne peut pas être attaché. Vous devez executer le service avec le drapeau 'it' ou bien utiliser `stdin_open: true, tty: true` dans votre fichier docker-compose.yaml",
-		WaitingForContainerInfo:    "Le processus ne peut pas continuer avant que docker donne plus d'informations. Veuillez réessayer dans quelques instants.",
+		UnattachableContainerError: "Le conteneur ne peut pas être attaché. Vous devez exécuter le service avec le drapeau 'it' ou bien utiliser `stdin_open: true, tty: true` dans votre fichier docker-compose.yaml",
+		WaitingForContainerInfo:    "Le processus ne peut pas continuer avant que docker ne donne plus d'informations. Veuillez réessayer dans quelques instants.",
 
 		CannotAttachStoppedContainerError: "Vous ne pouvez pas vous attacher à un conteneur en arrêt, vous devez le démarrer avant (ce que vous pouvez faire avec la touche 'r') (oui, je suis trop paresseux pour le faire automatiquement pour vous) (plutôt cool que je puisse communiquer en tête-à-tête avec vous au travers d'un message d'erreur)",
-		CannotAccessDockerSocketError:     "Impossible d'accèder à la socket docker à: unix:///var/run/docker.sock\nLancez lazydocker en tant que root ou alors lisez https://docs.docker.com/install/linux/linux-postinstall/",
-		CannotKillChildError:              "Trois secondes ont étés attendu pour l'arrêt des processus fils. Il y a possiblement un processus orphelin qui continue à tourner sur votre systeme.",
+		CannotAccessDockerSocketError:     "Impossible d'accéder à la socket docker à: unix:///var/run/docker.sock\nLancez lazydocker en tant que root ou alors lisez https://docs.docker.com/install/linux/linux-postinstall/",
```

Comment:
```suggestion
		CannotAccessDockerSocketError:     "Impossible d'accéder au socket docker à : unix:///var/run/docker.sock\nLancez lazydocker en tant que root ou lisez https://docs.docker.com/install/linux/linux-postinstall/",
```
Il me semble que parle généralement d'une soquette 🧦 mais d'_un_ socket. 😉